tests/unit/EntityMenu.spec.ts

```diff
@@ -214,17 +214,17 @@ describe('Entity Menu - View and Change Business Information click tests', () =>
 
     // set configurations
     configurationStore.setConfiguration(configuration)
+
```

Comment:
The following unit test changes are not related to this ticket whatsoever.

6 unit tests were failing from previous PR merges. I don't know why the GitHub CI didn't catch that or maybe we missed that the testing coverage was failing. Anyway, I fixed these tests.